python/ray/tests/test_placement_group.py

```diff
@@ -28,6 +28,24 @@ def method(self, x):
         return x + 2
 
 
+@pytest.mark.parametrize("connect_to_client", [True, False])
+def test_placement_ready(ray_start_regular_shared, connect_to_client):
+    @ray.remote
+    class Actor:
+        def __init__(self):
+            pass
+
+        def v(self):
+            return 10
+
+    with connect_to_client_or_not(connect_to_client):
+        pg = ray.util.placement_group(bundles=[{"CPU": 1}])
+        ray.get(pg.ready())
+        a = Actor.options(num_cpus=1, placement_group=pg).remote()
```

Comment:
This is not the right test isn't this? We should test with the default actor that requires 0 cpu?

Reply:
no, this PR only fix the issue https://github.com/ray-project/ray/issues/15842 here

Reply:
Hmm I see. Can you actually add a comment here to explain what the test is doing? I think you can say; 

```
# Test when all the bundle is occupied by the placement group, ready task is still schedulable
```

Also, I prefer to test 0 cpu case because that's the behavior change the PR will invoke.

Reply:
oh actually, I saw https://github.com/ray-project/ray/pull/16819#issuecomment-875842202. 

Does that mean this won't work for 0 cpu actor case still right?

Reply:
Last followup after sync:
- Comment what the test is doing. 
- Make the bundle resource private?